### PR TITLE
HMS-4247 - Adding iqe testing template for edge-frontend

### PR DESCRIPTION
**Pull request by adarshdubey-star**
# Description

This file add's a trigger for IQE CJI for the edge frontend testing.

What is it?
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of react-dom (17.0.2) is older than required version (18.0.0)
* Installed version of react-router-dom (5.3.4) is older than required version (6.0.0)
* Installed version of @patternfly/react-core (4.267.6) is older than required version (5.0.0)
* Installed version of react (17.0.2) is older than required version (18.0.0)
* Installed version of @redhat-cloud-services/frontend-components-config (4.7.3) is older than required version (6.0.0)
* Installed version of @patternfly/react-icons (4.93.3) is older than required version (5.0.0)

File: deploy/edge_stage_frontend_testing.yml [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  description: "Unique job name suffix"
+  generate: expression
+  from: "[a-z0-9]{6}"
+- name: IQE_IMAGE
+  description: "container image path for the iqe plugin"
+  value: quay.io/cloudservices/iqe-tests:edge
+- name: ENV_FOR_DYNACONF
+  value: stage_proxy
+- name: USE_BETA
+  value: "true"
```
Comment (akhil-jha) [inline deploy/edge_stage_frontend_testing.yml:137]: Will there be new features coming in to edge?

File: deploy/edge_stage_frontend_testing.yml [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+- name: SELENIUM_MEMORY_REQUEST
+  value: 1Gi
+- name: SELENIUM_JAVA_OPTS
+  value: ''
+- name: VNC_GEOMETRY
+  value: '1920x1080'
+- name: IQE_PARALLEL_ENABLED
+  value: "false"
+- name: SE_NODE_SESSION_TIMEOUT
+  value: "600"
```
Comment (akhil-jha) [inline deploy/edge_stage_frontend_testing.yml:183]: No new line error? 🤔

File: deploy/edge_stage_frontend_testing.yml (hunk 1 above)
Comment (adarshdubey-star) [reply deploy/edge_stage_frontend_testing.yml:137]: Nothing planned as of now

Comment (akhil-jha) [reply deploy/edge_stage_frontend_testing.yml:137]: beta tends to be a little unstable with the amount of other service changes impacting your tests. If there is nothing new planned, maybe don't use beta?

Comment (adarshdubey-star) [reply deploy/edge_stage_frontend_testing.yml:137]: We can overide this in the app-sre as well right or should I set it false here

Comment (adarshdubey-star) [reply deploy/edge_stage_frontend_testing.yml]: I have switched the flag to false

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of react-dom (17.0.2) is older than required version (18.0.0)
* Installed version of react-router-dom (5.3.4) is older than required version (6.0.0)
* Installed version of @patternfly/react-core (4.267.6) is older than required version (5.0.0)
* Installed version of react (17.0.2) is older than required version (18.0.0)
* Installed version of @redhat-cloud-services/frontend-components-config (4.7.3) is older than required version (6.0.0)
* Installed version of @patternfly/react-icons (4.93.3) is older than required version (5.0.0)

**Review by akhil-jha — APPROVED** [review]
lgtm

**Review by ezr-ondrej — APPROVED** [review]
Looks good to Akhil, so lets get that in ;)
